src/components/layout.js

```diff
@@ -25,13 +25,29 @@ export default function Layout({ jumbotron, children }) {
         }
         
         table {
-         width: 150%;
-         padding: 2rem;
-         background-color: ${theme.tableBackground};
+          width: 150%;
+          padding: 2rem;
+          background-color: ${theme.tableBackground};
         }
+        
         td {
-         text-align: left;
-         padding: .5rem;
+          text-align: left;
+          padding: .5rem;
+        }
+        
+        .caution-header {
```

Comment:
I'll likely want to rework how this works (using MDX and a React component for this), but this should be good for now.